InputOutput.py

```diff
@@ -160,7 +160,7 @@ def rawParseArgs(program, parser = None, no_args = False) :
             sys.exit(1)
 
         
```

Comment:
Yes, these lines
```
if len(args) == 0 : 
            parser.print_help(sys.stderr)
             sys.exit(1)
```
will lead to a call of the function printing the help message when there is no argument. However, what we need to do here is makes not only ``AlphaPeel`` works but also ``AlphaPeel -h`` and `AlphaPeel --help`` works. 

The original code assumes that all the command line input with only one argument must be the case that the argument is a file. It leads to the error since ``-h`` and ``--help`` are both the only argument in the command but they are definitely not files. So I made the change that argument inputs with ``-h`` and ``--help`` became the exception.

Reply:
OK, I see. But where are the `-h` or `-help` (note that Andrew was using `-help` type style of args not `--help`) actually getting called then?

Reply:
By default, ``argparse`` will take ``-h`` and ``--help`` as the command to print out the help message. It is written in line 1732 to line 1737 of ``argparse.py`` if you are interested.

So actually, it is ``-h`` and ``--help`` that are working.
Do I need to change ``--help`` to ``-help``?

Reply:
@XingerTang consistency is important. General Linux/UNIX style is `-h` and `--help` so short arg with one dash and long args with two dashes. I believe that Andrew used `-help` style, that is one dash and long arg form. Can you please check this. Since help is the most critical, I suggest that we support `-h`, `-help`, and `--help` to be on the safe side. Please implement this in the code AND in the documentation.

Reply:
Where can I check ``-help``? Is it for a specific package? But it can't work for any packages using ``tinyhouse`` as it disables all single arguments that are not files. 

I will do ``-h``, ``-help`` and ``--help`` later.

Reply:
> Where can I check -help? Is it for a specific package? But it can't work for any packages using tinyhouse as it disables all single arguments that are not files.

I am not sure we are on the same page here. We started with https://github.com/AlphaGenes/AlphaPeel/issues/14 since `AlphaPeel -h` threw an error. I guess this is, as you say, because tinyhouse disables all single arguments that are not files. Does your change then mean this will not longer be the case or will still be the case?

Reply:
Sorry for the ambiguity. Previously, ``tinyhouse`` disables all single arguments that are not files, and that's the reason why I am confused with the ``-help`` styles Andrew used since I cannot come up with where it could be used. 

And now under this pull request, we enable the ``-h`` and ``--help``, but still not ``-help``. I will work further to ensure all these three commands can work well.